gulpfile.js/paths.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 	componentsFile: 'components.json',
 	componentsFileJS: 'components.js',
 	components: ['components/**/*.js', '!components/index.js', '!components/**/*.min.js'],
+	themes: 'themes/*.css',
```

Comment:
Do you see the `.min.min.css` files as well?

```suggestion
	themes: ['themes/*.css', '!themes/*.min.css'],
```

Reply:
Have made this changes and committed the code again.